### ISSUE TITLE
New Version 1.1.0 not working

### ISSUE THREAD

**Issue by gp-slick-coder** [Testy/vscode-testyts-test-adapter#6]
- no tests are shown
- no error messages in the output

seams to be a loading issue.

Version 1.0.0 is working fine.

**Comment by Aboisier** [comment]
Huh, that's odd. Do you have a repo I can use to test this? Also, does it give you an error message?

**Comment by gp-slick-coder** [comment]
[testy-error.zip](https://github.com/Testy/vscode-testyts-test-adapter/files/4885481/testy-error.zip)

**Comment by keinPlan** [comment]
out folder of the extension is missing

**Comment by Aboisier** [comment]
Thanks for the feedback, and sorry for the inconveniences. I'm currently on vacation and won't have access to a computer, but I'll fix this when I get back.

**Comment by JannesMeyer** [comment]
Is there anything I can help with to fix this?

I am still getting the below error with the latest version:

```
[exthost] [error] Activating extension Testy.vscode-testyts-test-adapter failed due to an error:
[exthost] [error] Error: Cannot find module 'c:\Users\me\.vscode\extensions\testy.vscode-testyts-test-adapter-1.1.0\out\main.js'
Require stack:
- c:\Program Files\Microsoft VS Code\resources\app\out\vs\loader.js
- c:\Program Files\Microsoft VS Code\resources\app\out\bootstrap-amd.js
- c:\Program Files\Microsoft VS Code\resources\app\out\bootstrap-fork.js
	at Function.Module._resolveFilename (internal/modules/cjs/loader.js:844:17)
....
```

**Comment by danielmeza** [comment]
I had the same issue

**Comment by Aboisier** [comment]
Hi! I managed to reproduce and fix the problem. Thanks @keinPlan , that was indeed the issue! I automated the release with a GitHub action, but it seems like it was misconfigured. 

This should be fixed with version 1.1.1. I'll close this for now.

Sorry this took so long to fix!
